editor/src/dialog/dialogs/export_dialog.rs

```diff
@@ -61,7 +61,7 @@ impl PropertyHolder for Export {
 		];
 
 		let artboards = self.artboards.iter().map(|(&val, name)| (ExportBounds::Artboard(val), name.to_string()));
-		let mut export_area_options = vec![(ExportBounds::AllArtwork, "All Artwork".to_string())];
+		let mut export_area_options = vec![(ExportBounds::AllArtwork, "All Artwork".to_string()), (ExportBounds::Selection, "Selection".to_string())];
```

Comment:
The ideal behavior would be a contextual graying-out of the "Selection" entry when nothing is selected. That would require plumbing through the ability to gray out menu entries, but this might be a good opportunity to do that. A less-ideal alternative would be removing the "Selection" entry entirely, but add a TODO comment suggesting it be grayed out instead someday. Graying out provides a better UX because it allows users to know what would be available in other situations, and to avoid getting confused about how something isn't showing up like they remember seeing in the past without realizing it's contextually unavailable.